pypeit/core/coadd.py

```diff
@@ -2692,16 +2705,15 @@ def get_wave_bins(thismask_stack, waveimg_stack, wave_grid):
 
     Parameters
     ----------
-    thismask_stack : `numpy.ndarray`_
-        shape (nimgs, nspec, nspat)
-        Good pixel mask indicating which pixels are on slits
-
-    waveimg_stack : `numpy.ndarray`_
-        shape (nimgs, nspec, nspat)
-        Wavelength images for each image in the image stack
-
-    wave_grid : `numpy.ndarray`_
-        shape (ngrid)
+    thismask_stack (list):
```

Comment:
wrong doc style.
should be:

`thismask_stack: list`

same for most of the rest of this doc string

Reply:
fixed